pkg/validation/release_test.go

```diff
@@ -171,7 +172,7 @@ func TestValidateReleases(t *testing.T) {
 				},
 			},
 			output: []error{
-				errors.New("the name of an integration release must not contain '.' but found root.ocp-4.11"),
+				fmt.Errorf("root[ocp-4.11]: the release name is not a valid part of an image stream name: %w", fmt.Errorf("must not contain '.'")),
```

Comment:
Not terribly important, but since you'll need to change it: I think you can just inline the inner `fmt.Errorf()` in the message string of the outer one.

Reply:
Tried before my first commit.
I cannot make it work.